docs/api-reference/gltools/context.md

```diff
@@ -0,0 +1,149 @@
+# Context Management
+
+**gltools** provides functions to initialize, instrument and manipulate WebGL contexts.
+
+## Usage
+
+Create a WebGL context, autocreating a canvas
+```js
+import {createGLContext} from '@luma.gl/gltools';
+const gl = createGLContext(); // Prefers WebGL2 but falls back to WebGL1
+```
+
+Create a WebGL2 context.
+```js
+import {createGLContext} from '@luma.gl/gltools';
+const gl = createGLContext({
+  webgl1: false,
+  throwOnError: false
+});
+if (!gl) {
+  console.error('WebGL2 not supported');
+}
+```
+
+Polyfill a WebGL context with features available in extensions.
+```js
+import {polyfillContext} from '@luma.gl/gltools';
+
+const gl = canvas.createContext('webgl'); // A WebGL 1 context
+polyfillContext(gl);
+
+// Using extension via WebGL 2 API
+const vao = gl.createVertexArray();
+```
+
+Instrument an externally-created context to work with other luma.gl classes.
+```js
+import {instrumentGLContext} from '@luma.gl/gltools';
+import {Model} from '@luma.gl/engine';
+
+const gl = canvas.createContext('webgl');
+
+instrumentGLContext(gl);
+
+// Instrumentation ensures the context works with higher-level classes.
+const model = new Model(gl, options);
+```
+
+## Functions
+
+
```

Comment:
Nit: Run `yarn lint fix` on these to get rid of empty lines etc?

Reply:
This didn't actually do anything. I guess linting isn't set up for the docs. Fixed this one manually, and I'll look into automating it.